status/templates/status/statusTable.html

```diff
@@ -30,8 +30,7 @@
           "success" {% else %} "danger" {% endif %}>
           <td>
             {% if submission.grouper|show_detail:user %}
-              <a target="_blank"
```

Comment:
I want to open a discuss that, if we should open a new tab to view code?
If we resubmit again and again, chances are there are many new tabs on the browser , which is pretty annoying.
I don't know if disabling open new tab would be better.